### ISSUE TITLE
The Donation link continues to show without a checkmark after visitors have donated. 

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/events.backdropcms.org#40]
The problem is that the link continues to show without a checkmark after visitors have registered and donated. 

We have had several people comment in Zulip that they are not sure if their registration is complete or if their donation worked, because that link appears to be unfinished. 

By adding checkmarks after the first two items in the list, people are expecting a checkmark after the third. 

I recognize that might be difficult, in which case, can we move this item off the original checklist or somehow add some help text to indicate "Donate (unless you have already)"

I'm not sure, but the problem is that people are confused that step 3 appears incomplete, even after it has been completed. 

![image](https://github.com/backdrop-ops/events.backdropcms.org/assets/3144571/4223e1a0-8a7b-45ea-983b-d98c21869b1a)

---

Solution: 
<img width="418" alt="Screenshot 2023-09-23 at 5 36 45 PM" src="https://github.com/backdrop-ops/events.backdropcms.org/assets/397895/a9627127-95ab-4c99-8883-7f8fe31819b3">

**Comment by jenlampton** [comment]
I like the suggestion to add additional text to step 3, indicating that you should only make a donation if you have not already done so (for each event). I worry that text might be too long for one line though, so mybe we should add a  ? where the ✔️  usually goes, and display the suggestion on hover over the ?.

**Comment by jenlampton** [comment]
Comment from @nattywebdev on https://github.com/backdrop-ops/events.backdropcms.org/issues/37

> My experience was exactly as described by @stpaultim - left me uncertain about whether my donation had been accepted, and whether my registration was therefore not complete.

**Comment by jenlampton** [comment]
Comments from @izmeez on https://github.com/backdrop-ops/events.backdropcms.org/issues/37

> This is happening for me. It may be because I was unable to make a donation when registering and it was done separately by our accounting. Thus the registration and donation don't match.

> Maybe worse still, when I login now there is nothing to indicate I am registered and I do not see any messages to indicate what sessions I am interested in attending. It is as though the donation is a required step, that cannot be done separately or skipped.

**Comment by jenlampton** [comment]
The block with the next steps (showing that you are registered) should
already be on every page. That block should also have the links to submit a
session. I don't think it has a link to the schedule. Maybe we should add
that?

There is no way to select sessions, I don't think?

**Comment by nattywebdev** [comment]
The block that shows you are registered only implies it, by showing the 'next steps' - it is not explicit.

![image](https://github.com/backdrop-ops/events.backdropcms.org/assets/54537636/68611211-5f28-4596-a066-885bcef80b19)

It's pretty disappointing that we are not able to show a slicker registration system using our very own CMS. A simple UX check on the registration process might be a good plan!

**Comment by jenlampton** [comment]
> I never saw, and still don't see, the you are registered block. Mine may be an edge case because some one else submitted the payment.

It sounds like that other person is registered, and you are not. Right now we do not support people registering other people, but only because you may be the first one who has needed it! Something we should consider turning on for the next Backdrop Live event :) 

> It's pretty disappointing that we are not able to show a slicker registration system using our very own CMS. A simple UX check on the registration process might be a good plan!

The paypal registration process is only temporary, so we have not been working on any kind of integtration wtih PayPal. The long-term plan is to switch to using the SFC's stripe account (where we will have slick integration) but there have been some administrative holdups on the SFC side (out of our hands) that is preventing that from moving forward as quickly as we would like. 

It's not that we don't know that the UX needs work ;)

**Comment by nattywebdev** [comment]
Thanks for the feedback @jenlampton . I understand that the payment processing is temporary, but is there nothing that can be done to make it at least look like a new registrant is in fact registered? A little bit of adjustment using Rules for example - the kind of thing that we site builders rely on to make processes display correctly when a module doesn't work exactly like we want it. 
I only want to see Backdrop shown off in its best light - something that we can all agree to, I'm sure!

**Comment by stpaultim** [comment]
We have a fresh report in Zulip today from someone who just set up their account. 

"I’ve had this issue just now trying to register. Same exact problem. I have proof of my successful payment via PayPal, fortunately. But the backdrop LIVE site shows me as half-registered, so my account doesn’t exist."

I realize that the account portion of this is seperate, but I think the lack of confirmation regarding PayPal is making the issue more confusing to folks.

**Comment by jenlampton** [comment]
On my other site that uses this registration module we have a custom block that says "YOU ARE REGISTERED" and "YOU ARE NOT REGISTERED". We could do something like that here too, and get rid of the checkmarks entirely.

**Comment by jenlampton** [comment]
I have made two changes which I think will help this problem drastically.

1) I have moved the un-changing "make a donation" link from the "Register now" section down to the "Next steps" section, with the other links that don't get checkmarks.

2) I have also added another block with registration state, that we can place into the sidebar if the link moving is not enough. 

<img width="418" alt="Screenshot 2023-09-23 at 5 36 45 PM" src="https://github.com/backdrop-ops/events.backdropcms.org/assets/397895/58c11b20-51a7-4694-8ee8-3011eb1751a2">
